proto/admin/v1/tenant.proto

```diff
@@ -79,3 +83,19 @@ message TenantServiceRevokeResponse {
   // Tenant the tenant
   api.v1.Tenant tenant = 1;
 }
+
+// TenantServiceAddMemberRequest is the request payload for adding a member to a tenant
+message TenantServiceAddMemberRequest {
+  // Tenant ID to which the member will be added
+  string tenant_id = 1;
+  // Email of the member to add
+  string member_id = 2;
+  // Role to assign to the new member
+  api.v1.TenantRole role = 3;
+}
+
+// TenantServiceAddMemberResponse is the response payload for the add member request
+message TenantServiceAddMemberResponse {
+  // Confirmation of the addition
+  bool success = 1;
```

Comment:
I think we can assume success already from the status code of the response. Can be left empty. Or is there another reason this is necessary?

Reply:
@Honigeintopf please remove the bool flag